lib/routes/account.js

```diff
@@ -377,13 +396,15 @@ module.exports = function (
 
         customs.check(request, email, 'accountLogin')
           .then(readEmailRecord)
+          .then(checkSecurityHistory)
           .then(checkNumberOfActiveSessions)
           .then(createSessionToken)
           .then(createKeyFetchToken)
           .then(emitSyncLoginEvent)
           .then(sendVerifyAccountEmail)
           .then(sendNewDeviceLoginNotification)
           .then(sendVerifyLoginEmail)
+          .then(recordSecurityEvent)
```

Comment:
@seanmonstar do we need to block the response here on the security operation here? or can we do 'response' then 'record event' in the future?